languages/csharp/exercises/concept/nullability/Nullability.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+public static class Badge
+{
+    public static int ComputeWidthPx(string? text,
```

Comment:
Maybe:

```suggestion
    public static int WidthInPixels(string? text,
```

or

```suggestion
    public static int ComputeWidthInPixels(string? text,
```

or 

```suggestion
    public static int CalculateWidthInPixels(string? text,
```